### PR TITLE
Hsinyun

### PR DESCRIPTION
**Pull request by hsin-yunn**
修修
shop_return_record return_all api
shop_order add invoice_carrier_type
shop_product add weight_capacity_unit
ecpayController function callback_ecpay_inpay_order() change shop_order status

### CODE REVIEW

File: src/Resources/ShopShipTimeSetting.php [hunk 1]
```diff
@@ -16,19 +16,21 @@ public function toArray($request)
...
-        'type'       => $this->type,
-        'start_time' => $this->start_time,
-        'end_time'   => $this->end_time,
-        'max_count'  => $this->max_count,
+        'id'                     => $this->id,
+        'type'                   => $this->type,
+        'start_time'             => $this->start_time,
+        'end_time'               => $this->end_time,
+        'max_count'              => $this->max_count,
+        'tpday_shop_order_count' => $this->today_shop_orders->count(),
```
Comment (davidturtle) [inline src/Resources/ShopShipTimeSetting.php:24]: today?

File: src/Resources/ShopShipTimeSetting.php [hunk 2]
```diff
@@ -16,19 +16,21 @@ public function toArray($request)
...
```
Comment (davidturtle) [inline src/Resources/ShopShipTimeSetting.php:33]: ""

File: src/Resources/ShopShipTimeSetting.php (hunk 1 above)
Comment (hsin-yunn) [reply src/Resources/ShopShipTimeSetting.php:24]: done 改成today
火眼晶晶

File: src/Resources/ShopShipTimeSetting.php (hunk 2 above)
Comment (hsin-yunn) [reply src/Resources/ShopShipTimeSetting.php:33]: done 改成today
火眼晶晶

**Pull request merged**